File: RV_JAVA/app/src/main/java/com/example/rv_java/MainActivity.java
package com.example.rv_java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import com.example.rv_java.prohect1.Activity1;
import com.example.rv_java.project2.Activity2;
import com.example.rv_java.project3.Activity3;

public class MainActivity extends AppCompatActivity {
    TextView tv1, tv2, tv3;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tv1 = findViewById(R.id.tv1);
        tv2 = findViewById(R.id.tv2);
        tv3 = findViewById(R.id.tv3);

        tv1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, Activity1.class));
            }
        });

        tv2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, Activity2.class));
            }
        });

        tv3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, Activity3.class));
            }
        });
    }
}

File: RecyclerView - KOTLIN/MyApplication/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    lateinit var et_usr:EditText
    lateinit var et_pass:EditText
    lateinit var btn_login:Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        et_usr = findViewById(R.id.et_usr)
        et_pass = findViewById(R.id.et_pass)
        btn_login = findViewById(R.id.btn_login)

        btn_login.setOnClickListener{
            if(!et_usr.text.toString().isNullOrEmpty() && !et_pass.text.toString().isNullOrEmpty()){
                var intent:Intent = Intent(this, SecondActivity::class.java)
                intent.putExtra("usr", et_usr.text.toString())
                startActivity(intent)
            }

        }

    }
}